modules/dynamic-plugin-api.adoc

```diff
@@ -1611,6 +1611,49 @@ const Component: React.FC = (props) => {
 .Returns
 A tuple containing the current active namespace and setter callback.
 
+[discrete]
+== `useUserSettings`
+
+Hook that provides a user setting value and a callback for setting the user setting value.
+
+.Example
+[source,tsx,subs="+quotes,+macros"]
+----
+const Component: React.FC = (props) ++=>++ {
+   const [state, setState, loaded] = useUserSettings(
+     ++'devconsole.addPage.showDetails'++,
+     true,
+     true,
+   );
+   return loaded ++? (++
+      ++<WrappedComponent {...props} userSettingState={state} setUserSettingState={setState} />++
+    ) : null;
+};
+----
+
+.Returns
+A tuple containing the user setting vauel, a setter callback, and a loaded boolean.
+
+[discrete]
+== `useQuickStartContext`
+
+Hook that provides the current quick start context values. This allows plugins to interop with console quick start functionality.
```

Comment:
interoperate? I think "quick-start" might also be a compound adjective here too, technically